### PR TITLE
Drawing reduce

### PR DESCRIPTION
**Pull request by BillyBarbaro**
Got rid of like 200 lines of code by rearranging the drawing code so we didn't set registers repeatedly.  Don't know if it's clean enough for you though.  Let me know.  Even if you reject the request, keep the branch.

### CODE REVIEW

File: pong.asm [hunk 1]
```diff
@@ -21,435 +21,286 @@
 NewGame:
 	jal ClearBoard
 
-	leftLines:
-		li $a0, 10 #the x starting coordinate
-		li $a1, 13 #the y coordinate
-		lw $a2, colorTwo #the color
-		li $a3, 18 #the x ending coordinate
+	Lines:
+		li $a0, 10
```
Comment (AndrewHamm) [inline pong.asm:25]: @BillyBarbaro How come you got rid of the comments for x-coord and y-coord?

**Comment by AndrewHamm** [comment]
@BillyBarbaro I like this! The only comment I have is that it got rid of lots of the labels and comments saying where stuff is happening which makes it harder to read.

File: pong.asm (hunk 1 above)
Comment (BillyBarbaro) [inline pong.asm:25]: The arguments are annotated where the method is declared.  So I figured that was sufficient.

Comment (AndrewHamm) [inline pong.asm:25]: @BillyBarbaro Yeah I just looked at the file by itself and I agree with you. I like it. Argument revoked.

**Pull request merged**